content/enterprise_influxdb/v1.9/features/_index.md

```diff
@@ -9,4 +9,72 @@ menu:
     weight: 60
 ---
 
-{{< children hlevel="h2" >}}
+InfluxDB Enterprise has additional capabilities that enhance
+[availability](#clustering),
+[scalability](#clustering), and
+[security](#security),
+as well as providing [eventual consistency](#eventual-consistency).
+
+## Clustering
+
+InfluxDB Enterprise runs on a network of independent servers, a *cluster*,
+to provide fault tolerance, availability, and horizontal scalability of the database.
+
+Many InfluxDB Enterprise features are available
+when run with a single meta node and a single data node
+However this does not take advantage of the clustering capablity,
+and therefore does not provide high availablity assurances.
+
+Nodes can be added to an existing cluster to improve database performance for querying and writing data.
+Certain configurations (e.g., 3 meta and 2 data node) provide high-availability assurances
+while making certain tradeoffs in query peformance when compared to a single node.
+
+Further increasing the number of nodes can improve performance in both respects
+For example, a cluster with 4 data nodes and a [replication factor]() of 2
+can support a higher volume of write traffic than a single node could.
+It can also support a higher *query* workload, as the data is replicated
+in two locations. Performance of the queries may be on par with a single
+node in cases where the query can be answered directly by the node which
+receives the query.
+
+For more information on clustering, see [Clustering in InfluxDB Enterprise](/enterprise_influxdb/v1.9/concepts/clustering/).
+
+## Security
+
+Enterprise authorization uses an expanded set of [*16 user permissions and roles*](/enterprise_influxdb/v1.9/features/users/).
+(InfluxDB OSS only has `READ` and `WRITE` permissions.)
+Administrators can give users permission to read and write to databases,
+create and remove databases, rebalance a cluster, and manage particular resources.
```

Comment:
this covers the conceptual overview of permissions--we probably don't need a separate doc (and can keep the Configure/Manage sections strictly procedural content